packages/fx-core/src/plugins/resource/identity/constants.ts

```diff
@@ -41,3 +41,18 @@ export class Telemetry {
     appid: "appid",
   };
 }
+export class IdentityBicep {
```

Comment:
`IdentityBicep` is not very accurate and concrete. What about `OutputBicepSnippet`?